website/data/cloud-docs-nav-data.json

```diff
@@ -134,6 +134,7 @@
       { "title": "Subscriptions", "path": "api-docs/subscriptions" },
       { "title": "Team Membership", "path": "api-docs/team-members" },
       { "title": "Team Tokens", "path": "api-docs/team-tokens" },
+      { "title": "Organization Team Tokens", "path": "api-docs/organization-team-tokens" },
```

Comment:
Instead of creating a separate page for this, I do think we should either add this to the [Team token page](https://developer.hashicorp.com/terraform/cloud-docs/api-docs/team-tokens) or honestly I think the [main organization API page](https://developer.hashicorp.com/terraform/cloud-docs/api-docs/organizations) would be a good fit too. Since it's only one endpoint I think it could fit well on either page.